device-types/Fortinet/FSM-2000F.yaml

```diff
@@ -0,0 +1,24 @@
+---
+manufacturer: Fortinet
+model: FortiSIEM 2000F
+slug: fortinet-fsm-2000f
+part_number: FSM-2000F
+u_height: 2
+is_full_depth: false
+console-ports:
+  - name: Console
+    type: de-9
+power-ports:
+  - name: PS1
+    type: iec-60320-c14
+  - name: PS21
```

Comment:
Is this actually suppose to be called `PS21` or did you mean for it to be `PS2`?